apps/user/views.py

```diff
@@ -12,6 +10,21 @@ class UserViewSet(viewsets.ModelViewSet):
     queryset = User.objects.all()
     serializer_class = UserResponseSerializer
 
+    def list(self, request, *args, **kwargs):
+        chat_id = self.request.query_params.get('chat_id')
+        keywords = self.request.query_params.get('keywords')
+        univ = self.request.query_params.get('univ')
+        if chat_id:
+            queryset = User.objects.filter(chat_id=chat_id).all()
+            response_serializer = UserResponseSerializer(queryset, many=True)
+            return Response(data=response_serializer.data, status=status.HTTP_200_OK)
+        elif keywords and univ:
+            queryset = User.objects.filter(keywords=keywords, univ=univ).all()
+            response_serializer = UserResponseSerializer(queryset, many=True)
+            return Response(data=response_serializer.data, status=status.HTTP_200_OK)
+        else:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
요즘 쿼리빌더 형식으로 작성해보고 있는데 아래처럼 작성하면 훨씬 깔끔한듯용! 

```suggestion
        queryset = self.get_queryset()
        if chat_id:
            queryset = queryset.filter(chat_id=chat_id)
        elif keywords and univ:
            queryset = queryset.filter(keywords=keywords, univ=univ)
        else:
            return Response(status.status.HTTP_400_BAD_REQUEST)
        users = queryset.all()
        response_serializer = UserResponseSerializer(users, many=True)
        return Response(data=response_serializer.data, status=status.HTTP_200_OK)
```